group_project_v2/project_api/api_implementation.py

```diff
@@ -391,3 +394,24 @@ def get_member_data(self, user_id):
         if user_organizations:
             user_details.organization = user_organizations[0]['display_name']  # and a string here
         return user_details
+
+    @memoize_with_expiration()
+    def get_organisation_by_id(self, org_id):
```

Comment:
@jbzdak nit: check spelling of organization - there are british spelling "organisation" and american spelling "organization". Please use one of those, not both :)